modules/analyzing-the-collector-pod-status.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: PROCEDURE
+[id="analyzing-the-collector-pod-status_{context}"]
+= Analyzing the Collector pod status
+
+Examining the pod’s most recent status is another easy way to determine the cause of a Collector crash. Failure messages are recorded to the most recent status and are accessible using the `kubectl` or `oc` command:
```

Comment:
using the `kubectl describe pod` or `oc describe pod` command

Reply:
ack